deno/sync-with-meilisearch/src/main.ts

```diff
@@ -0,0 +1,78 @@
+import {
+  Client,
+  Databases,
+  Query,
+} from "https://deno.land/x/appwrite@9.0.0/mod.ts";
+import { getStaticFile, interpolate, throwIfMissing } from "./utils.ts";
+import { MeiliSearch } from "https://esm.sh/meilisearch@0.35.0";
+
+export default async ({ req, res, log, error }: any) => {
+  throwIfMissing(Deno.env.toObject(), [
+    "APPWRITE_API_KEY",
+    "APPWRITE_DATABASE_ID",
+    "APPWRITE_COLLECTION_ID",
+    "MEILISEARCH_ENDPOINT",
+    "MEILISEARCH_INDEX_NAME",
+    "MEILISEARCH_ADMIN_API_KEY",
+    "MEILISEARCH_SEARCH_API_KEY",
+  ]);
+
+  if (req.method === "GET") {
+    const html = interpolate(await getStaticFile("index.html"), {
+      MEILISEARCH_ENDPOINT: Deno.env.get("MEILISEARCH_ENDPOINT"),
+      MEILISEARCH_INDEX_NAME: Deno.env.get("MEILISEARCH_INDEX_NAME"),
+      MEILISEARCH_SEARCH_API_KEY: Deno.env.get("MEILISEARCH_SEARCH_API_KEY"),
+    });
+
+    return new res.send(html, 200, {
+      "Content-Type": "text/html; charset=utf-8",
+    });
+  }
+
+  const client = new Client()
+    .setEndpoint(
+      Deno.env.get("APPWRITE_ENDPOINT") ?? "https://cloud.appwrite.io/v1",
```

Comment:
Is it possible to do Deno.env.get("APPWRITE_ENDPOINT")!! ?

If so could be better.

Reply:
it is not possible unfortunately.